geopolars/src/geoseries.rs

```diff
@@ -5,17 +5,28 @@ use arrow2::array::{
     ArrayRef, BinaryArray, BooleanArray, MutableBinaryArray, MutableBooleanArray,
     MutablePrimitiveArray, PrimitiveArray,
 };
-use geo::{Geometry, Point};
+use geo::algorithm::affine_ops::AffineTransform;
+use geo::{map_coords::MapCoords, Geometry, Point};
 use geozero::{CoordDimensions, ToWkb};
 use polars::prelude::{PolarsError, Result, Series};
+use std::convert::Into;
 
 pub enum GeodesicLengthMethod {
     Haversine,
     Geodesic,
     Vincenty,
 }
 
+pub enum TransformOrigin {
+    Centroid,
+    Center,
```

Comment:
It might be nice to have docstrings on these enum values. I wouldn't understand at a glance what the difference is between `Centroid` and `Center`. 

It looks like `Center` means "take the bounding rectangle and then take the center/centroid of that rectangle"? Would something like `BoundingRectCenter` be clearer as the enum value?

Reply:
Added these